src/js/controllers/tab-home.controller.js

```diff
@@ -67,6 +69,10 @@ angular
     $scope.$on("$ionicView.afterEnter", onAfterEnter);
     $scope.$on("$ionicView.leave", onLeave);
 
+    moonPayService.getUserByIpAddress().then(function(user) {
+      isBuyBitcoinAllowed = user.isAllowed;
```

Comment:
This is good enough for this release. It could be an issue though, for people wanting to quickly navigate through the app, if the request takes a while.